frontend/src/js/components/header/DeviceNotifications.tsx

```diff
@@ -11,86 +11,235 @@
+
+  return (
+    <div className={className}>
+      <Tooltip
+        title="View device limits"
+        open={!anchorEl && tooltip}
+        disableFocusListener
+        disableTouchListener
+        disableHoverListener={!!anchorEl}
+        onMouseEnter={() => !anchorEl && setTooltip(true)}
+        onMouseLeave={() => setTooltip(false)}
+      >
+        <div>
+          <Badge color={severity} badgeContent={pending} anchorOrigin={{ horizontal: 'left' }}>
+            <Button
+              startIcon={<DeveloperBoardIcon className="margin-right-x-small" fontSize="small" />}
+              endIcon={<ArrowDropDownIcon fontSize="small" sx={{ color: 'text.secondary' }} className={classes.endIcon} />}
+              className={`flexbox center-aligned ${classes[severity]} ${!!anchorEl && classes.pressed}`}
+              onClick={handleOpen}
+            >
+              <span>{total.toLocaleString('en-US')}</span>
+            </Button>
+          </Badge>
+          <Popover
+            onClose={handleClose}
+            anchorEl={anchorEl}
+            open={Boolean(anchorEl)}
+            anchorOrigin={{
+              vertical: 'bottom',
+              horizontal: 'right'
+            }}
+            transformOrigin={{
+              vertical: 'top',
+              horizontal: 'right'
+            }}
+            slotProps={{ paper: { className: classes.popover } }}
+          >
+            {!!pending && (
+              <Alert
+                severity="info"
+                className="flexbox center-aligned margin-bottom-small"
+                onClick={() => viewPendingClick()}
+                slotProps={{ message: { className: 'flexbox center-aligned space-between full-width' } }}
+              >
+                <Typography variant="body2">
+                  {pending} {pluralize('device', pending)} pending
+                </Typography>
+                <Button className="padding-bottom-none padding-top-none" size="small" variant="text">
+                  View
+                </Button>
+              </Alert>
+            )}
+            <div className="margin-bottom-x-small">
+              <div className="flexbox space-between full-width">
+                <div className="flexbox centered">
+                  <DeveloperBoardIcon className="margin-right-x-small" fontSize="small" />
+                  <Typography variant="subtitle1">Accepted devices</Typography>
+                </div>
+                <Typography variant="subtitle1" color="text.secondary">
+                  {total.toLocaleString('en-US')}
+                </Typography>
+              </div>
+            </div>
+            <Divider />
+            <div className="margin-top-small">
+              {mappedLimits.map(limit => (
+                <DeviceLimit key={limit.type} {...limit} />
+              ))}
+            </div>
+            {isAdmin && (
+              <Button color="primary" variant="text" onClick={() => onManageDeviceLimit()}>
```

Comment:
to be more semantically correct I think:

```suggestion
              <Button color="primary" variant="text" component={Link} to="/settings/subscription" onClick={handleClose}>
```

_should_ also work, but the `onManageDeviceLimit` is also ok